server/app/controllers/api/v1/wishes_controller.rb

```diff
@@ -1,12 +1,13 @@
 require 'uri'
+require 'pry'
 class Api::V1::WishesController < ApplicationController
+  skip_before_action :authorize_request, only: [:show, :index] 
   before_action :set_api_user
   before_action :set_api_user_wish, only: [:show, :update, :destroy]
 
-
   #GET /api_users/:api_user_id/wishes
   def index
-    @wishes = @api_user.wishes.includes(:programminglanguage, :meetinginterval)  if params[:api_user_id]
+    @wishes = @api_user.wishes.includes(:programminglanguage, :meetinginterval) # if params[:api_user_id]
```

Comment:
Is this commented out code meant to be deleted?

Reply:
Yes, removed now